ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.BrayfloxsLongstop,
+  triggers: [
+    {
+      // Numbing Breath is Stunnable
+      id: 'Brayflox Normal Numbing Breath',
+      netRegex: NetRegexes.startsUsing({ id: '1FA', source: 'Great Yellow Pelican' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Pelican Adds
+      id: 'Brayflox Normal Pelican Adds',
+      netRegex: NetRegexes.message({ line: 'A flock of ziz emerges from the jungle!', capture: false }),
+      response: Responses.killAdds('info'),
+    },
+    {
+      // Ashdrake Cyclone Stun
+      id: 'Brayflox Normal Ashdrake Burning Cyclone',
+      netRegex: NetRegexes.startsUsing({ id: '205', source: 'Ashdrake' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Tempest Biast Spawn
+      id: 'Brayflox Normal Tempest Biast',
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '1285', capture: false }),
+      response: Responses.killAdds('info'),
+    },
+    {
+      // Inferno Drake Cyclone Stun
+      id: 'Brayflox Normal Inferno Drake Burning Cyclone',
+      netRegex: NetRegexes.startsUsing({ id: '3D8', source: 'Inferno Drake' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Hellbender Bubble Others
+      id: 'Brayflox Normal Hellbender Effluvium Others',
+      netRegex: NetRegexes.startsUsing({ id: '3D3', source: 'Hellbender' }),
+      condition: function(data, matches) {
+        return matches.target != data.me;

```

Comment:
In general, prefer using `===` and `!==` instead of `==` and `!=` (here and elsewhere in this patch).  In this case, you can also say `condition: Conditions.targetIsNotYou(),`